controllers/topolvm_snapshotClass.go

```diff
@@ -0,0 +1,103 @@
+package controllers
```

Comment:
- there seems to be no filename with camelcase, so it's better to use only small letters in filenames?

Reply:
done.